packages/utilities/psammead-assets/README.md

```diff
@@ -43,6 +43,22 @@ const WrappingContainer = () => (
 );
 ```
 
+## Media Icons SVGs
+
+Media icons is an object containing styled SVG icons for video, audio and photogallery. Media icons are sized to work well alongside specific text with typography group `GEL Minion`. They are used in `psammead-media-indicator` component.
+
+## Usage
+
+```jsx
+<MediaIndicatorWrapper>
```

Comment:
Thanks for adding an example. It'd be good to also include the import and make it as simple as possible - so excluding the wrappers & other styled components.
```jsx
import mediaIcons from '@bbc/psammead-assets/mediaIcons';
...
<span>
  {mediaIcons.audio}
  {duration && datetime && (<time dateTime={datetime}>{duration}</time>)}
</span>
```

Reply:
Thank you! Done :) Though the path for the import that I added is a bit different since `mediaIcons` is in the `svgs`